index.html

```diff
@@ -853,66 +868,227 @@ <h2 id="BaseAudioContext">
               be played to the destination, and then allows the system to
               release its claim on audio hardware. This is generally useful
               when the application knows it will not need the
-              <a>BaseAudioContext</a> for some time, and wishes to let the
-              audio hardware power down. The promise resolves when the frame
+              <a>BaseAudioContext</a> for some time, and wishes to temporarily
+              <a>release system resource</a> associated with te
+              <a>BaseAudioContext</a>. The promise resolves when the frame
               buffer is empty (has been handed off to the hardware), or
               immediately (with no other effect) if the context is already
-              suspended. The promise is rejected if the context has been
-              closed.
+              <code>"uspended"</code>. The promise is rejected if the context
+              has been closed.
+            </p>
+            <p>
+              <span class="synchronous">When suspend is called, execute these
+              steps:</span>
+            </p>
+            <ol>
+              <li>Let <em>promise</em> be a new Promise.
+              </li>
+              <li>If the <em>control thread state</em> flag on the
+              <a>AudioContext</a> is <code>closed</code> reject the promise
+              with <code>InvalidStateError</code>, abort these steps, returning
+              <em>promise</em>.
+              </li>
+              <li>If the <a href="#widl-AudioContext-state">state</a> attribute
+              of the <a>AudioContext</a> is already <code>suspended"</code>,
+              resolve <em>promise</em>, return it, and abort these steps.
+              </li>
+              <li>Set the <em>control thread state</em> flag on the
+              <a>AudioContext</a> to <code>suspended</code>.
+              </li>
+              <li>
+                <a href="#queue">Queue a control message</a> to suspend the
+                <a>AudioContext</a>.
+              </li>
+              <li>Return <em>promise</em>.
+              </li>
+            </ol>
+            <p>
+              Running a <a>control message</a> to suspend an
+              <a>AudioContext</a> means running these steps on the <a>rendering
+              thread</a>:
             </p>
+            <ol>
+              <li>Attempt to <a>release system resources</a>.
```

Comment:
I'll make it so that it's required to release resources when calling `close()`, and that implementation can choose to release some resources when doing `suspend()`. Tracing some code in chrome, I see that there is at least a thread being released.